samples/kubernetes/end2end/docs/existing-domain.md

```diff
@@ -0,0 +1,36 @@
+## Monitoring an Existing WebLogic Domain
+In this step, we show you how to integrate Prometheus and Grafana with an existing WebLogic domain.
+
+Assume you have already followed [Manage Domains](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/) guide in the WebLogic Kubernetes Operator to run a WebLogic domain and have followed previous steps in this sample to setup Prometheus and Grafana.
```

Comment:
Assume you have already followed [Manage Domains] guide -> We assume that you have already followed the [Manage Domains] guide
and have followed previous steps in this sample to setup Prometheus and Grafana -> and have followed the previous steps in this sample to set up Prometheus and Grafana